README.md

```diff
@@ -160,6 +160,13 @@ set -ag status-right "#{E:@catppuccin_status_session}"
 set -ag status-right "#{E:@catppuccin_status_uptime}"
 set -agF status-right "#{E:@catppuccin_status_battery}"
 
+# automatically set CWD as window name
```

Comment:
Why is `#W` the current directory?

Reply:
I took this from https://github.com/catppuccin/tmux/issues/53#issuecomment-1675211611. But I'm not knowledgeable enough to say what it means.